core/src/test/scala/org/apache/spark/storage/BlockManagerDecommissionIntegrationSuite.scala

```diff
@@ -190,7 +190,8 @@ class BlockManagerDecommissionIntegrationSuite extends SparkFunSuite with LocalS
     logInfo(s"Decommissioning executor ${execToDecommission}")
     sched.decommissionExecutor(
       execToDecommission,
-      ExecutorDecommissionInfo("", isHostDecommissioned = false))
+      ExecutorDecommissionInfo("", isHostDecommissioned = false),
+      adjustTargetNumExecutors = true)
```

Comment:
Why is this true and not false ? We explicitly want to kill and discard the executor here without replacing it. Although the test does not truly care, but still why the change ?

Reply:
Same please reread the code

Reply:
^^^ @holdenk ... any thoughts/followup on this ?

Reply:
Same.

Reply:
This is also resolved. Reasoning sounds good -- although do you want to add the same comment as above: 

// Decommission executor and ensure it is not relaunched by setting adjustTargetNumExecutors